src/test/javascript/spec/service/faq.service.spec.ts

```
@@ -0,0 +1,209 @@
+                .subscribe((resp) => (expectedResult = resp));
+            const req = httpMock.expectOne({
+                url: `api/courses/${courseId}/faqs/${faqId}`,
+                method: 'PUT',
+            });
+            req.flush(returnedFromService);
+            expect(expectedResult.body).toEqual(expected);
+        });
+
+        it('should delete a faq', () => {
+            const returnedFromService = { ...elemDefault };
+            const faqId = elemDefault.id!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)